index.html

```diff
@@ -64,7 +64,7 @@ <h4 class="h4-connect">LET'S CONNECT</h4>
 
```

Comment:
### Your Logo
![image](https://user-images.githubusercontent.com/93039359/195176430-74426a0d-8814-4838-8003-9ac01e8a44ea.png)

- [ ] Your logo doesn't have a 4.5:1 color contrast ratio. Kindly make it to follow the requrements and to make your portfolio accessible to color blind users :blush:

Reply:
This task is done. We actually used a darker purple that can make it more visible.